doc/reactive-migration.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: Extensions.Migration.Reactive
+---
+
+# Upgrading Extensions Reactive Version
+
+## Upgrading to Extesions Reactive V5
+
+Uno.Extension.Reactive V5 is a major release that includes a number of breaking changes. The following guide will help you upgrade your application from V4 to V5.
+
+### MVUX Generated Code
+
+As of V4, the MVUX analyzer generated a bindable proxy for each of the models in your app. For example, the bindable proxy for `MainModel` was named `BindableMainModel`.
+
+This behavior changed in V5, and the MVUX analyzer will now generate a ViewModel class for each of your app's models. For example, the generated code for `MainModel` is named `MainViewModel`.
+
+To upgrade your application, you will need to update the references to the generated classes. These are usually referenced in the code-behind file when using XAML, for example, `MainPage.xaml.cs`, or in the Page UI definition when using C# for Markup, for example, `MainPage.cs`.
```

Comment:
Don't we need to also document the need for the assembly attribute to use the new generators?

Reply:
This is partially correct. Your comment made me recall that migrating to the latest will not require any changes as we set version 2 (the previous version) as the default to avoid breaking changes. The above is only necessary if we manually set version 3; for that, we use the assembly attribute. 

I need to reword this documentation. Thanks @kazo0

Reply:
Fixed the wording. Thanks!